s3_website.yml

```diff
@@ -0,0 +1,59 @@
+s3_id: <%= ENV['AWS_ACCESS_KEY'] %>
+s3_secret: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+s3_bucket: adb-mongolia-nua
```

Comment:
Good idea. I'll switch it to use an env variable and we can set that up once we have other buckets to deploy to.